main.py

```diff
@@ -11,12 +16,12 @@ def main():
     try:
         connection_parameters = parse_connection_arguments()
     except RuntimeError as ex:
```

Comment:
Remove `as ex`, because it is no more used

Reply:
resolved